doc/integrator/objectstorage.rst

```diff
@@ -0,0 +1,109 @@
+.. _integrator_objectstorage:
+
+Working with Object storage (like S3)
+=====================================
+
+Generalities
+------------
+
+In this we will consider to use the S3 like from Exoscale.
+
+First of all we should set the following variable on the ``geoportal``, ``config`` and ``qgisserver`` services.
+
+* ``AWS_ACCESS_KEY_ID``: The project access key.
+* ``AWS_SECRET_ACCESS_KEY``: The project secret key.
+* ``AWS_DEFAULT_REGION=ch-dk-2``: The region use by Exoscale.
+* ``AWS_S3_ENDPOINT=sos-ch-dk-2.exo.io``: The endpoint used by Exoscale.
+
+And for performances we should set the following variable on the ``geoportal`` and ``qgisserver`` services.
```

Comment:
"And for performances we should..." -> "For better performance, we should..."
"variable" -> "variables"